framework/build.gradle

```diff
@@ -20,6 +20,12 @@ ext {
     apply from: 'cordova.gradle'
     cdvCompileSdkVersion = privateHelpers.getProjectTarget()
     cdvBuildToolsVersion = privateHelpers.findLatestInstalledBuildTools()
+    if (project.hasProperty('cdvMinSdkVersion')) {
+        cdvMinSdkVersion = Integer.parseInt('' + cdvMinSdkVersion)
+        println "[Cordova] cdvMinSdkVersion is overridden, try it at your own risk."
```

Comment:
The log statement should use single quotes.

Reply:
Changed it to single quotes. Easy. 
There are although other places printing with double quotes. https://github.com/apache/cordova-android/blob/master/framework/cordova.gradle#L53